setup/static/render.js

```diff
@@ -1,21 +1,30 @@
 //Store API token in variable.
 var token = $("#api_token").val();
 
+$(document).ready(function() {
+	$("#no_java_script").hide();
+	$("#container").show();
+});
+
 $(document).ready(function() {
 	if ($("#webmail").val() == 'none') {
 		$("#webmail_path").hide();
 		$("#webmail_path").val("");
+		$("#webmail_path").prop('required',false);
```

Comment:
I would rather get rid of all ``WEB_`` variables tbh. We agreed Mailu should have it's own FQDN, let's simplify and hard-code paths on it.

Reply:
I'm fine with removing it from setup only. Setup should be simplified and help the end-user not making any mistakes. A typical user should not have to change the `WEB_` settings. 

Advanced users can still tweak mailu.env.